loom/engine/loom2d/l2dScript.cpp

```diff
@@ -67,7 +67,7 @@ bool Loom2DNative::sInitialized = false;
 
 static Matrix *StaticMatrixConstructor(lua_State *L)
 {
-    return new Matrix((float)lua_tonumber(L, 2), (float)lua_tonumber(L, 3),
+    return lmNew(NULL) Matrix((float)lua_tonumber(L, 2), (float)lua_tonumber(L, 3),
```

Comment:
Did these only work before because we call new/delete in the bowels of lmNew/lmDelete later?